src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala

```diff
@@ -33,6 +33,7 @@ object SchemaConverter {
   val typeMap = Map(
     "string" -> StringType,
     "number" -> DoubleType,
+    "float" -> FloatType,
```

Comment:
cool, thank you. @AndreiDev do you need the new release published any time soon?